backend/src/routes/__test__/encounter.route.test.ts

```diff
@@ -378,6 +388,54 @@ describe('POST /encounter', () => {
     })
 })
 
+describe('GET /encounters/:id', () => {
+    it ('Successfully retrieves an encounter with a given json', async () => {
+        const newEncounterId = await createUserPersonEncounter(token);
+
+        // retrieve the specified encounter
+        const encounter = await supertest(app)
+            .get(`/api/encounters/${newEncounterId}`)
+            .set('Accept', 'application/json')
+            .set('Authorization', token)
+            .expect(httpStatus.OK)
+
+        expect(encounter._body.description).toBe(encounterData.description)
+        expect(encounter._body.location).toEqual(encounterData.location)
+
+    });
+  
+    it('Fails when called with invalid encounter object ID', async () => {
+        await createUserPersonEncounter(token)
+        // update an encounter of id that does not exist
+        await supertest(app)
+            .get(`/api/encounters/${123}`)
+            .set('Accept', 'application/json')
+            .set('Authorization', token)
+            .expect(httpStatus.NOT_FOUND)
+    });
+
+    it('Fails when encounter object ID that does not exist', async () => {
```

Comment:
This may be a duplicate test as the one above already passed in an objectID string that does not match with any encounters.